### ISSUE TITLE
Color contrast in css Code for light mode

### ISSUE THREAD

**Issue by straker** [dequelabs/cauldron#417]
CSS syntax has some issues for attribute selector

```
<Code language="css">a[href="mars2\.html\?a\=be_bold"] > h3</Code>
```

![image](https://user-images.githubusercontent.com/2433219/141368948-3f5586b2-e448-4e8e-b9d2-27dd4caa91a7.png)

Axe report: 

```
"Fix any of the following:
  Element has insufficient color contrast of 2.03 (foreground color: #f99157, background color: #f0f2f5, font size: 11.3pt (15px), font weight: normal). Expected contrast ratio of 4.5:1"
```

**Comment by scurker** [comment]
@straker Was this in the latest extension? We've made changes but they haven't been released yet.

**Comment by dequejosie** [comment]
This issue has been fixed. Here is the new look and it has no color-contrast issue.
![Uploading Screen Shot 2021-12-14 at 11.21.46 AM.png…]()
